### DISCUSSION TITLE
Is it possible to know when a user registers a model via GUI?

### DISCUSSION THREAD

**Discussion by andre-cantcode-fabiao** [mlflow/mlflow#7522] [Q&A] [answered]
Hi guys, I am not very experienced with mlflow so sorry if this question is trivial.

I wanted to know if there is any way to know when a user registers a model via the GUI, and retrieve which model is being registered, time of registry, that type of information.

Thanks in advance!

**Comment by JHibbard** [comment, marked as answer]
Hi andre-cantcode-fabiao,

I don't believe there's a way to find this information through the UI. The UI attempts to show the most relevant information on any particular page but you'll have to use one of the programmatic APIs to access the full metadata associated with an MLflow record.

Hope that helps!
Jim
